src/java/org/apache/cassandra/db/commitlog/CommitLogSegmentManagerCDC.java

```diff
@@ -84,21 +84,46 @@ public void discard(CommitLogSegment segment, boolean delete)
 
     /**
      * Delete the oldest hard-linked CDC commit log segment to free up space.
+     * @param bytesToFree, the minimum space to free up
      * @return total deleted file size in bytes
```

Comment:
I think we either need to amend this javadoc @return to match that we're returning the bytes remaining or we need to juggle around variable names below. Currently this seems incorrect.

It looks like we're expecting CDC bytes remaining in CommitLogSegmentManagerCDC.processNewSegment, so I think this is probably just a vestigial javadoc entry.

Reply:
I forgot to address this one. I will update the doc soon